### DISCUSSION TITLE
Domain is pending verification

### DISCUSSION THREAD

**Discussion by Kaung-Htet-Naing** [vercel/community#3131] [Help] [answered]
### Summary

I recently deleted my old project connect with the domain after that i created a new repo and i still getting this error

**Comment by amyegan** [comment, marked as answer]
Hi @Kaung-Htet-Naing. Sometimes domain ownership verification is required to protect a domain from being hijacked. If you add the `TXT` record with your registrar where you added the `A` and `CNAME` records for the domain, this message will go away and your domain will be connected to the new project.

**Reply by nguyentienanh59** [reply]
please let me ask, what's the A and CNAME? i have the same problem

**Reply by GabrielDiasFaria** [reply]
Go to this page: https://vercel.com/docs/projects/domains/managing-dns-records
